File: test/fixtures/node-got/post_xpost.js
import got from 'got';

const response = await got.post('http://localhost:28139/api/xxxxxxxxxxxxxxxx', {
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  body: '{"keywords":"php","page":1,"searchMode":1}'
});

File: test/fixtures/ruby-httparty/post_xpost.rb
require 'httparty'

url = 'http://localhost:28139/api/xxxxxxxxxxxxxxxx'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = '{"keywords":"php","page":1,"searchMode":1}'
res = HTTParty.post(url, headers: headers, body: body)
